### DISCUSSION TITLE
[OKD 4.16] Option to change default 15m for deploy initial nodes  in vSphere mode?

### DISCUSSION THREAD

**Discussion by rasystems1** [okd-project/okd#2051] [Q&A]
Hello team, there is a method to can change the default 15 minutes about deploy initial nodes (bootstrap and master) in openshift-installer? I tried to deploy OKD 4.16 in vSphere mode but in 15 minutes not finish the initial installation, the internet performance is not very high so when bootstrap download initial image,  is about 5/8 minutes to the first reboot and change to SCOS and in the rest of the time the masters nodes not complete the deploy

**Comment by jcpowermac** [comment]
I would suggest using a disconnected install if your internet performance is poor. There is no configuration to extend bootstrap time.

You can run `openshift-install wait-for bootstrap-complete` and `openshift-install wait-for install-complete` to retry _but_ I would suggest using disconnected first.
